### ISSUE TITLE
Debuggin stack

### ISSUE THREAD

**Issue by marioles50** [rtlabs-com/p-net#587]
Is it possible to debug the p-net stack with Netbeans?
If so, how can I do it?
or with other type Eclipse Visual studio code etc
thanks

**Comment by elupus** [comment]
Yes. You debug it as any standard embedded software or standard system. Load the built elf file in your chosen debugger and set up symbol paths if needed.
